src/main/java/pianokeys/Note.java

```diff
@@ -0,0 +1,8 @@
+package pianokeys;
+
+public record Note()
+{
+    public static int key;
```

Comment:
I will remind you that the use of static variables will cause you to fail the class.

Please look up how to properly use records.